### ISSUE TITLE
add support for iris motion sensor v3 

### ISSUE THREAD

**Issue by redwoodimage** [dresden-elektronik/deconz-rest-plugin#2460]
Manufacturer: iMagic by GreatStar
Product Name: Device
Model Number: 1117-S
deviceTypeId: 15
manufacturer:iMagic by GreatStar
address64bit:D0CF5EFFFEC8D68B

Here's a links with more info on the sensor https://community.hubitat.com/t/new-iris-motion-sensor-version-generation-3/5190/2

![007](https://user-images.githubusercontent.com/39349413/74613159-df636f00-50c0-11ea-8bd5-780ff1df7fe5.png)
![006](https://user-images.githubusercontent.com/39349413/74613162-e0949c00-50c0-11ea-9ee6-24717a04f2a2.png)
![005](https://user-images.githubusercontent.com/39349413/74613163-e0949c00-50c0-11ea-91c0-d6c9fb3e581e.png)
![004](https://user-images.githubusercontent.com/39349413/74613164-e12d3280-50c0-11ea-83d1-4f71169fc094.png)
![003](https://user-images.githubusercontent.com/39349413/74613165-e1c5c900-50c0-11ea-9757-970aaf2e1ca9.png)
![002](https://user-images.githubusercontent.com/39349413/74613166-e1c5c900-50c0-11ea-9e1b-a0e47a13ac95.png)
![001](https://user-images.githubusercontent.com/39349413/74613168-e25e5f80-50c0-11ea-851b-2bab741da82a.png)

**Comment by rmotapar** [comment]
Just wanted to say that I am seeing the same issue as well when adding iris motion sensor v3. It shows up in the mesh network but does not show up in the webapp. I do see an open pull request  for this so that's good news!

![Centralite iL07_1 motion sensor](https://user-images.githubusercontent.com/13870280/74991839-bd455600-540c-11ea-9010-672923915c30.PNG)

**Comment by SwoopX** [comment]
The device will not show up in Phoscon, as the software is closed source. DE must handle integration there.

**Comment by rmotapar** [comment]
@SwoopX  Thank you! I assumed it would show up in Phoscon app. Are there any other ways to get this device integrated with home assistant when using Conbee ii?

**Comment by SwoopX** [comment]
When the new deconz version gets release, the missing sensors should be created automatically and therefore show up in HA. If not, just try to repair.

Meanwhile, you could add the missing bindings manually for power configuration, temperature and humidity cluster. Regarding motion, I'm not sure if the IAS Zone cluster is in enrolled status. If not, you can also enroll it manually by using the MAC address of your Conbee. Search the issues for binding guidance (its really not that difficult) and for manual enrollment, that has been explained in one of the heiman CO2 sensor issues.

**Comment by rmotapar** [comment]
@SwoopX  Thank you for the response! I was able to add the iris motion sensor v3. Temperature and humidity values started showing up in homeassistant after pairing but as you suspected there was nothing related to motion. Thanks for the pointers on how to enroll them manually, I was able to enroll the IAS Zone cluster and motion events are now showing up. Thanks again! 

I am wondering what it would take to do the same for Iris motion sensors V2. They are made by centralite with model number 3326-L. I have a bunch of these and would like to integrate them into homeassistant. There is an existing thread related to these sensors here - https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1731.
